apps/client/lib/client/repeatable_lists/create_list_from_template.ex

```diff
@@ -22,56 +23,60 @@ defmodule Client.RepeatableLists.CreateListFromTemplate do
 
       {:ok, _result} =
         template
-        |> item_attrs(list)
-        |> Enum.reduce(Ecto.Multi.new(), fn item_cset, multi ->
-          Ecto.Multi.insert(multi, :rand.uniform(), item_cset)
+        |> section_attrs(list)
```

Comment:
flipping order of inserts here. We set a Section association on Items, so the sections need to have been inserted already otherwise the foreign key constraint will fail.